erpnext/selling/doctype/customer/test_customer.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
This looks like the default template. How about update it or remove it completely.